build-tools/script/util/getPages.js

```diff
@@ -11,9 +11,9 @@ const resolvers = {
   yaml: path => yaml.safeLoad(fs.readFileSync(path, 'utf8')),
 };
 
-const replaceVariables = loadData(path.resolve(projectRoot, 'src/data/_variables.yaml'), {
+const replaceVariables = ((path) => fs.existsSync(path) && loadData(path, {
```

Comment:
you're shadowing the `path` variable, better to use a different parameter name in case you need to use the actual `path` module inside there :D